standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -2956,6 +2956,7 @@ PartitionsResponse get_partitions_req(1:PartitionsRequest req)
   void mark_compacted(1: CompactionInfoStruct cr) throws(1:MetaException o1)
   void mark_failed(1: CompactionInfoStruct cr) throws(1:MetaException o1)
   void mark_refused(1: CompactionInfoStruct cr) throws(1:MetaException o1)
+  void retry_cleaner_attempt_with_backoff(1: CompactionInfoStruct cr, 2:i64 retentionTime) throws(1:MetaException o1)
```

Comment:
I don't think this is necessary, since the Cleaner runs only in HMS and can communicate directly with the CompactionTxnHandler.

Reply:
Ok I removed it.